CHANGELOG.rst

```diff
@@ -5,6 +5,12 @@ Change history for XBlock
 Unreleased
 ----------
 
+5.1.0 - 2024-08-07
```

Comment:
Thanks for the changelog entry. Could you update the docs too? Maybe just a quick "Replace a preinstalled XBlock with a custom implementation" guide under https://github.com/openedx/XBlock/tree/master/docs/xblock-tutorial/edx_platform (which [publishes here](https://docs.openedx.org/projects/xblock/en/latest/xblock-tutorial/edx_platform/index.html)) ?

The docs in that directory are admittedly a mess (although we [have a volunteer working on them](https://github.com/openedx/XBlock/issues/774) :), so don't feel like you need to write anything stellar.

Reply:
@kdmccormick , will tackle in a follow-up PR.

Reply:
JK, didn't realize this was the same repo. Will add it shortly.

Reply:
@kdmccormick , docs updated here: https://github.com/openedx/XBlock/pull/778/commits/5720d4a934e0cafb834d8c85a0b7641bb1f49811